pyat/at/plot/resonances.py

```diff
@@ -0,0 +1,505 @@
+    "label": "15n",
+}
+prop1s = {
+    "color": mypalettecolor[1],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(4, WIDTHMOD),
+    "label": "1s",
+}
+prop2s = {
+    "color": mypalettecolor[2],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(3, WIDTHMOD),
+    "label": "2s",
+}
+prop3s = {
+    "color": mypalettecolor[3],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(2, WIDTHMOD),
+    "label": "3s",
+}
+prop4s = {
+    "color": mypalettecolor[4],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(2, WIDTHMOD),
+    "label": "4s",
+}
+prop5s = {
+    "color": mypalettecolor[5],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "5s",
+}
+prop6s = {
+    "color": mypalettecolor[6],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "6s",
+}
+prop7s = {
+    "color": mypalettecolor[7],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "7s",
+}
+prop8s = {
+    "color": mypalettecolor[8],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "8s",
+}
+prop9s = {
+    "color": mypalettecolor[9],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "9s",
+}
+prop10s = {
+    "color": mypalettecolor[10],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "10s",
+}
+prop11s = {
+    "color": mypalettecolor[11],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "11s",
+}
+prop12s = {
+    "color": mypalettecolor[12],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "12s",
+}
+prop13s = {
+    "color": mypalettecolor[13],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "13s",
+}
+prop14s = {
+    "color": mypalettecolor[14],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "14s",
+}
+prop15s = {
+    "color": mypalettecolor[15],
+    "linestyle": mypalettestyle["skew"],
+    "linewidth": numpy.mod(1, WIDTHMOD),
+    "label": "15s",
+}
+# assemble default dictionary by normal or skew
+propn = {
+    1: prop1n,
+    2: prop2n,
+    3: prop3n,
+    4: prop4n,
+    5: prop5n,
+    6: prop6n,
+    7: prop7n,
+    8: prop8n,
+    9: prop9n,
+    10: prop10n,
+    11: prop11n,
+    12: prop12n,
+    13: prop13n,
+    14: prop14n,
+    15: prop15n,
+}
+props = {
+    1: prop1s,
+    2: prop2s,
+    3: prop3s,
+    4: prop4s,
+    5: prop5s,
+    6: prop6s,
+    7: prop7s,
+    8: prop8s,
+    9: prop9s,
+    10: prop10s,
+    11: prop11s,
+    12: prop12s,
+    13: prop13s,
+    14: prop14s,
+    15: prop15s,
+}
+# assemble final dict
+defaultlprop = {0: propn, 1: props}
```

Comment:
Still: I think it would be more clear to have
```python
defaultlprop = {"normal": propn, "skew": props}
```

Reply:
Done.